app/assets/stylesheets/finder_frontend.scss

```diff
@@ -197,6 +197,11 @@
 
   .js-enabled & {
     display: inline-block;
+    &:focus {
+      -webkit-box-shadow: inset 0 0 0 2px;
+      box-shadow: inset 0 0 0 2px;
+      border: 1px solid #000;
+    }
```

Comment:
Tiny nitpick: when the element is focused the 'x' moves very slightly, presumably related to the border addition? If you set a transparent 1px border on the default state it seems to fix it.

Normal:

<img width="551" alt="Screen Shot 2019-09-06 at 16 01 59" src="https://user-images.githubusercontent.com/861310/64438421-fb012d00-d0bf-11e9-99b6-e9278a2af6ba.png">

Focus:

<img width="551" alt="Screen Shot 2019-09-06 at 16 02 08" src="https://user-images.githubusercontent.com/861310/64438432-005e7780-d0c0-11e9-9fc9-5666a579383d.png">

Actually it's easier to see this not as screenshots, oh well.

Reply:
yeah, I spotted it moved, but wasn't too bothered. 
have added the transparent border now